packages/core/quri_parts/core/estimator/gradient.py

```diff
@@ -171,13 +177,17 @@ def parameter_shift_gradient_estimates(
     estimates_dict = dict(zip(gate_params_list, estimates))
 
     grad = []
+    err_diag = []
     for params_and_coefs in shifted_params_and_coefs:
         g = 0.0 + 0.0j
+        var = 0.0
         for p, c in params_and_coefs:
             g += estimates_dict[p].value * c
+            var += (estimates_dict[p].error ** 2) * abs(c)
```

Comment:
I guess $|c|$ should also be squared:
```suggestion
            var += (estimates_dict[p].error ** 2) * abs(c) ** 2
```